app/src/main/res/values/strings.xml

```diff
@@ -89,7 +89,8 @@
   <string name="chapter_name">Chapter %s: %s</string>
   <string name="chapter_completed">Chapter %s with title %s is completed</string>
   <string name="chapter_in_progress">Chapter %s with title %s is in progress</string>
-  <string name="chapter_prerequisite_title_label">Chapter %s: %s is currently locked. Please complete chapter %s: %s to unlock this chapter.</string>
+  <string name="chapter_prerequisite_title_label">Complete Chapter %s: %s to unlock this chapter.</string>
+  <string name="chapter_lock_prerequisite_title_label">Chapter %s: %s is currently locked. Please complete chapter %s: %s to unlock this chapter.</string>
```

Comment:
did you intend this as an adjective meaning the chapter is locked? In which case it could be `chapter_locked_prerequisite_title_label`

Reply:
Done.